### ISSUE TITLE
Validate whitespace around <<

### ISSUE THREAD

**Issue by daviddavis** [bbatsov/rubocop#1651]
Rubocop seems not to mind this code:

```ruby
a = []
a                   <<                  '1'
```

I'd expect SpaceAroundOperators to pick this up.

**Comment by bquorning** [comment]
This was a surprise to me, too. It seems that `Style::SpaceAroundOperators` only complains if there are *no* spaces around the operators, not if there are too many.

**Comment by bbatsov** [comment]
Pretty sure the reason for this was `=` and the fact that some people write code like:

```
ala_bala = 5
x             = 10
```

I think we should start checking for extra whitespace and just introduce a config option to ignore certain ops.

**Comment by bquorning** [comment]
That makes sense.

It looks like it's pretty easy to change `SpaceAroundOperators` to check for too many spaces. Should I try fixing this one?

**Comment by bbatsov** [comment]
You definitely should! :-)

**Comment by daviddavis** [comment]
:+1: @bbatsov and @bquorning

**Comment by bquorning** [comment]
Proposed fix in #1654.
